functests/5_latency_testing/latency_testing.go

```diff
@@ -105,12 +110,12 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 				testlog.Error(err.Error())
 			}
 			Expect(string(output)).NotTo(MatchRegexp(unexpectedError), "Unexpected error was detected in a positve test")
-			//Check runtime argument in the pod's log only if the tool is expecetd to be executed
-			res, err := regexp.MatchString(success, string(output))
+			//Check runtime argument in the pod's log only if the tool is expected to be executed
+			ok, err := regexp.MatchString(success, string(output))
 			if err != nil {
-				testlog.Error(err.Error())
+				testlog.Info(err.Error())
```

Comment:
why `Error` -> `Info`

Reply:
Well..Because the error would give us information on why the parsing failed which will help in debugging only.
Do you believe we better keep it as Error? (if so, I will consider changing the logging for similar errors)

Reply:
It depends on the purpose of the message (this is true in general).
As rule of thumb:

- If the condition is expected to pass, so the failure is exceptional, use Error
- If the condition can pass or fail, and the failure is expected and recoverable, use Info